nerdlets/e2m-gui-nerdlet/form-components/section2.js

```diff
@@ -1,146 +1,140 @@
+          >
+            Name and describe your new rule
+          </HeadingText>
+        </div>
+        <div className={`body ${activeStep === 2 ? 'expand' : 'collapse'}`}>
+          <div className="section-heading">Metric name</div>
+          <div className="section-description">
+            Your metric name must be unique. We recommend using dot separation
+            between sections of the name.
+          </div>
+          <MetricNameValidator
+            ref={nameTextfield}
+            selectedAccountID={selectedAccountID}
+            selectedEventType={selectedEventType}
+            selectedAttribute={selectedAttribute}
+            selectedAggregator={selectedAggregator}
+            selectedFacetAttributes={selectedFacetAttributes}
+            validatedFilterNRQL={validatedFilterNRQL}
+            setValidatedMetricName={validatedMetricName =>
+              setValidatedMetricName(validatedMetricName)
+            }
+            e2mRulesByMetricName={e2mRulesByMetricName}
+            cardinalityDataLoading={cardinalityDataLoading}
+          />
+          <div className="section-heading">Rule alias</div>
+          <RuleAliasValidator
+            ref={aliasTextfield}
+            selectedAccountID={selectedAccountID}
+            setValidatedRuleAlias={validatedRuleAlias =>
+              setValidatedRuleAlias(validatedRuleAlias)
+            }
+            selectedEventType={selectedEventType}
+            selectedAttribute={selectedAttribute}
+            selectedAggregator={selectedAggregator}
+            selectedFacetAttributes={selectedFacetAttributes}
+            validatedFilterNRQL={validatedFilterNRQL}
+            e2mRulesByMetricName={e2mRulesByMetricName}
+            cardinalityDataLoading={cardinalityDataLoading}
+          />
+          <div className="section-heading">Rule description</div>
+          <RuleDescriptionValidator
+            ref={descriptionTextfield}
+            e2mRulesByMetricName={e2mRulesByMetricName}
+            selectedAccountID={selectedAccountID}
+            selectedEventType={selectedEventType}
+            selectedAttribute={selectedAttribute}
+            selectedAggregator={selectedAggregator}
+            selectedFacetAttributes={selectedFacetAttributes}
+            validatedFilterNRQL={validatedFilterNRQL}
+            setValidatedDescription={validatedRuleDescription =>
+              setValidatedDescription(validatedRuleDescription)
+            }
+            cardinalityDataLoading={cardinalityDataLoading}
+          />
+          <Button className="next" onClick={() => setActiveStep(1)}>
+            Previous Step
+          </Button>
+          {!readyForStepThree() ? null : (
+            <Button
+              className="next"
+              onClick={() => setActiveStep(3)}
+              type={Button.TYPE.PRIMARY}
+            >
+              Next Step
+            </Button>
+          )}
+        </div>
+      </div>
+    </div>
+  );
 };
```

Comment:
Have to stop checking here. Another meeting.

Reply:
This one is pretty straightforward formatting cleanup.